css/creative.css

```diff
@@ -408,6 +402,49 @@ aside {
   }
 }
 
+@media (max-width: 991px) {
+  .help-img2 {
+    margin: 0 215px;
+  }
```

Comment:
Some of the help related classes in `@media (max-width: 991px)` can be moved back to index.css. Make a `@media (max-width: 991px)` and put some of those help classes back in.